src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V> [] table;
+    private int size;
+    private int capacity;
+    private int threshold;
```

Comment:
current threshold and capacity can be calculated based on size and array length, not need to have separate variables for them
```suggestion
```